Marlin/Marlin_main.cpp

```diff
@@ -52,6 +52,8 @@
  * G4   - Dwell S<seconds> or P<milliseconds>
  * G5   - Cubic B-spline with XYZE destination and IJPQ offsets
  * G10  - Retract filament according to settings of M207 (Requires FWRETRACT)
+ *        G10 S1 for ToolChange long retractation 
+ *        Zlift feedrate according to setting of M203 (Feedrate limits)
```

Comment:
Move this kind of documentation to the comment above `gcode_G10`. These descriptions are intentionally terse.